PROJECT

```diff
@@ -34,4 +34,13 @@ resources:
   kind: DRPlacementControl
   path: github.com/ramendr/ramen/api/v1alpha1
   version: v1alpha1
+- api:
+    crdVersion: v1
+    namespaced: true
```

Comment:
We think DRCluster should be scoped similar to DRPolicy and also because it is a representation of a managed cluster, it should not be namespaced.

Reply:
I had/have reservations on the scope, and wanted to be sure we want this at cluster scope and not namespaced:
- Do we need users of the cluster to be able to see the resource?
- Can a cluster scoped resource (DRPolicy) point to a namespaced resource (DRCluster)?

I have initially chosen a namespace scope, to see what problems we may encounter during implementation (do not see any yet other than when we attempt to list all DRClusters.

Anyway long story short, I did not want to apply ManagedCluster scope rules to this one, but wanted more reasons for its scope to be at the cluster level, and I am still debating what is correct.